modules/services/compose/src/main/java/au/com/shiftyjelly/pocketcasts/compose/components/GradientIcon.kt

```diff
@@ -56,6 +58,43 @@ fun GradientIcon(
     )
 }
 
+@Composable
+fun GradientIcon(
+    painter: Painter,
+    gradientBrush: Brush,
+    contentDescription: String? = null,
+    blendMode: BlendMode = BlendMode.SrcAtop,
+    modifier: Modifier = Modifier,
+) {
+    Icon(
+        modifier = modifier
+            .graphicsLayer(alpha = 0.99f)
```

Comment:
Why's this `alpha` needed?

Reply:
If I remove this alpha the icon on top looks like this 
<img width="155" alt="image" src="https://github.com/user-attachments/assets/f1a1cc2f-8b0b-4d26-bbd6-10b4188c01e9">

This was an solution that I found for gradient

Reply:
Got it, thanks!